src/data/prayers/__tests__/data-source.test.js

```diff
@@ -45,4 +45,9 @@ describe('PrayerRequest data sources', () => {
     expect(await cursor.get()).toMatchSnapshot();
     expect(await cursor.count()).toMatchSnapshot();
   });
+  it('answers a prayer', async () => {
+    Prayer.patch = () => null;
```

Comment:
Is this just trying to define a function that returns `null`? Would using `jest.fn()` here be incorrect?

Reply:
we don't need to mock the function unless we need to see inside the call